openshift_tools/ircbot/trello/trello.py

```diff
@@ -128,6 +150,97 @@ def get(self):
                                                  members)
         return results
 
+    def report(self, email=None, move=False):
```

Comment:
I wasn't able to find a sopel command that can run a report. Is that intended? It might be useful to run a report on demand? Although I can't say that I know the intended use case =)

Reply:
This is CLI only right now. Depending on how the use case settles out, I suspect we'll run a timed sopel cmd. Not sure yet.